app/src/main/resources/application.properties

```diff
@@ -110,3 +110,25 @@ quarkus.http.cors=true
 # Disable OpenAPI class scanning
 mp.openapi.scan.disable=true
 mp.openapi.servers=/api
+
+# The ignored error list
+%dev.registry.liveness.errors.ignored=${LIVENESS_ERRORS_IGNORED:io.grpc.StatusRuntimeException,org.apache.kafka.streams.errors.InvalidStateStoreException}
```

Comment:
We should add this property to the test `application.properties`. That's the reason for the `verify` workflow to fail.